clicommand/annotate.go

```diff
@@ -97,8 +97,9 @@ var AnnotateCommand = cli.Command{
 		},
 		cli.IntFlag{
 			Name:   "priority",
-			Usage:  "Priority of the annotation (1 to 10). By default annotations have a priority of 3. Annotations with a priority of 10 will be shown first, and annotations with a priority of 1 will be shown last.",
+			Usage:  "The priority of the annotation (′1′ to ′10′). Annotations with a priority of ′10′ are shown first, while annotations with a priority of ′1′ are shown last. When this option is not specified, annotations have a default priority of ′3′.",
```

Comment:
With `Value: 3` below, you can probably drop the part about "When this option is not specified..." since it should be clear through the usual mechanisms that `3` is actually the default.

Reply:
Thanks @DrJosh9000 , good point, and I'm assuming `(default: 3)` will be shown in the docs (looking at other examples in the Agent code and Buildkite Docs).